<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Kaushik Renganaath" />


<title>SNP based COTA pilot results</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">my_project_notes</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">SNP based COTA pilot results</h1>
<h4 class="author">Kaushik Renganaath</h4>
<h4 class="date">11 April, 2025</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-11
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>my_project_notes/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed2024code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(2024)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed2024code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(2024)</code> was run prior to running the
code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomrenga011workingnotesliulabtreef1cf6cf8562fadd68b684bf67345d6182c6f265btargetblankf1cf6cfa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/renga011/working_notes_liu_lab/tree/f1cf6cf8562fadd68b684bf67345d6182c6f265b" target="_blank">f1cf6cf</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomrenga011workingnotesliulabtreef1cf6cf8562fadd68b684bf67345d6182c6f265btargetblankf1cf6cfa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/renga011/working_notes_liu_lab/tree/f1cf6cf8562fadd68b684bf67345d6182c6f265b" target="_blank">f1cf6cf</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/Gene_based_COTA/
    Ignored:    analysis/SNP_based_COTA/.DS_Store

Unstaged changes:
    Modified:   analysis/2024-04-10_snp_summary.Rmd
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/2024-04-10_snp_summary.Rmd</code>) and HTML
(<code>docs/2024-04-10_snp_summary.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/renga011/working_notes_liu_lab/b5dee20a19ec4ae2736217e5562daedf296e9145/docs/2024-04-10_snp_summary.html" target="_blank">b5dee20</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/renga011/working_notes_liu_lab/b646a9ea6a65d28c62539ff64339cf8f78b7d99a/docs/2024-04-10_snp_summary.html" target="_blank">b646a9e</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/renga011/working_notes_liu_lab/blob/e432b0ce9e58e816d90546533f383b45dacd09f9/analysis/2024-04-10_snp_summary.Rmd" target="_blank">e432b0c</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
<td>
wflow_publish("analysis/2024-04-10_snp_summary.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="cota-results-with-lee-et-al.-2023" class="section level2">
<h2>COTA results with Lee et al. (2023)</h2>
<p>Using:</p>
<ul>
<li><p><strong>Cruchaga trans pQTL</strong> (renamed genes)</p></li>
<li><p><strong>WES burden test</strong> scores from Lee et al.</p></li>
</ul>
<blockquote>
<p><strong>some statistics of the WES data from Lee et al.</strong></p>
<ul>
<li><p>Size of study cohort (ADSP 17k): 6519 cases (total 13371 with
controls)</p></li>
<li><p>Burden scores were computed for genes using different sets of
variant types. Number of genes for each of these variant categories are
as follows</p>
<ul>
<li><p>plof_ds (putative loss of function - deleterious strict) -
15306</p>
<blockquote>
<p>WE USE THESE GENES FOR COTA AS THESE ARE HIGH CONFIDENCE, HIGH
IMPACT</p>
</blockquote></li>
<li><p>plof - 11771</p></li>
<li><p>disruptive_missense - 10776</p></li>
<li><p>missense - 18170</p></li>
<li><p>synonymous - 18100</p></li>
</ul></li>
</ul>
</blockquote>
<div id="result-summary" class="section level3">
<h3>Result summary</h3>
<ul>
<li><p><strong># proteins tested in trans pQTL</strong>: 6983</p></li>
<li><p><strong># proteins common with WES data (using only
pLOF_ds)</strong>: 5530</p></li>
<li><p><strong># significant pairs detected</strong>:
<em>0</em></p></li>
<li><p>No detected significant pairs from COTA passed the FDR threshold
of <em>q &lt; 0.1</em>, despite the sample size being ~6.5k cases (total
~13k with controls).</p></li>
</ul>
</div>
<div id="validity-of-wes-p-values" class="section level3">
<h3>Validity of WES P-values</h3>
<p>We checked the distribution of WES-derived <em>p</em>-values:</p>
<blockquote>
<p><strong>Insert QQ plot and histogram here</strong></p>
</blockquote>
<p><img src="figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-1-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-1">
Past versions of unnamed-chunk-1-1.png
</button>
</p>
<div id="fig-unnamed-chunk-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/renga011/working_notes_liu_lab/blob/b646a9ea6a65d28c62539ff64339cf8f78b7d99a/docs/figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-1-1.png" target="_blank">b646a9e</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>The qq-plot shows a small kink away from the null in between,
which maybe due to this being p values pooled across ancestries, a
smaller effective sample size (lesser carriers), or combination of
both.</p></li>
<li><p>~ This is only 6k samples and pooled across ancestries. Maybe
therefore, there is no much power in the COTA</p></li>
</ul>
</div>
<div id="what-next" class="section level3">
<h3>What next?</h3>
<ul>
<li>Transition to Holstege et al. study data: Sample size ~16k cases
(Total size with controls ~32.5k), and its all mostly Non hispanic
white</li>
</ul>
<hr />
</div>
</div>
<div id="cota-results-with-holstege-data" class="section level2">
<h2>COTA Results with Holstege data</h2>
<div id="validity-of-wes-p-values-1" class="section level3">
<h3>Validity of WES p values</h3>
<p><img src="figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-2-1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/renga011/working_notes_liu_lab/blob/b646a9ea6a65d28c62539ff64339cf8f78b7d99a/docs/figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-2-1.png" target="_blank">b646a9e</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cota-run---notes" class="section level3">
<h3>COTA run - notes</h3>
<blockquote>
<p>Gene tagging with multiple aptamers</p>
<p>The trans pQTL data from Cruchaga has multiple aptamers tagging the
same gene/protein.</p>
<ul>
<li>Correlation between aptamers tagging same protein in Cruchaga et
al.: <strong>0.79</strong></li>
</ul>
<p>To preserve this signal:</p>
<ul>
<li><p>When gene IDs are duplicated, we rename them as
<code>GENE_1</code>, <code>GENE_2</code>, etc.</p></li>
<li><p>If even one aptamer shows significance, it’s worth examining as
it may provide clues on the mechanism (eg: If aptamer specific to
N-terminus is significantly mediating but that targetted to C-terminus
isn’t).</p></li>
</ul>
<p>So we apply this renaming consistently across:</p>
<ul>
<li><p><code>pQTL_trans: p value matrix for the trans eQTLs</code></p></li>
<li><p><code>WES_pvals: p values corresponding to Holstege rare variant burden scores</code></p></li>
<li><p><code>ref.table.keep: annotations for every gene on its start, end, chromosome, gene name, gene type (protein_coding/lncRNA)</code></p></li>
<li><p><code>pQTL_cis: cis pQTLs detected by Cruchaga et al. (2023). Taken from their Supplementary tables</code></p></li>
</ul>
</blockquote>
<p>Using:</p>
<ul>
<li><p><strong>Cruchaga trans pQTL</strong> (renamed genes)</p></li>
<li><p><strong>WES burden test</strong> scores from Holstege et
al.</p></li>
</ul>
<blockquote>
<p><strong>some statistics of the WES data Holstege et al.</strong></p>
<ul>
<li>Size of study cohort (mixed cohort - see Supplementary table 1 of
paper) : 6519 cases (total 13371 with controls)</li>
</ul>
<p>Burden scores were computed for genes using different sets of variant
types. Number of genes for each of these variant categories are as
follows</p>
<ul>
<li><p>LOF and REVEL score &gt;= 25 - 15306</p>
<blockquote>
<p>WE USE THESE GENES FOR COTA as most genes are included in the data
for this category</p>
</blockquote></li>
<li><p>LOF and REVEL score &gt;= 50 - 11771</p></li>
<li><p>LOF and REVEL score &gt;= 75 - 10776</p></li>
</ul>
</blockquote>
</div>
<div id="result-summary-1" class="section level3">
<h3>Result Summary:</h3>
<ul>
<li><p><strong># proteins tested in trans pQTL</strong>: 6983</p></li>
<li><p><strong># proteins common with WES data</strong>: 4855</p></li>
<li><p><strong># genes in the WES data with significant burden test
score (p &lt; 1e-6): 4</strong></p></li>
<li><p><strong># significant pairs detected (q &lt; 0.1):
210</strong></p></li>
<li><p><strong>Core genes: 7</strong></p></li>
<li><p><strong>Peripheral genes: 18</strong></p></li>
<li><p>Gene-dense regions are more informative than individual gene
counts.</p></li>
</ul>
<blockquote>
<p>genomic regions with hits: 4</p>
</blockquote>
</div>
<div id="network-diagram" class="section level3">
<h3>Network Diagram</h3>
<p><img src="figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-3-1.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/renga011/working_notes_liu_lab/blob/b646a9ea6a65d28c62539ff64339cf8f78b7d99a/docs/figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-3-1.png" target="_blank">b646a9e</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="pathway-enrichment" class="section level3">
<h3>Pathway Enrichment</h3>
<blockquote>
<p>Perform GO/KEGG/Reactome enrichment.</p>
</blockquote>
<ul>
<li><p>No significant enrichments for core genes. Mostly due to their
only being 7.</p></li>
<li><p>The following core genes were also burden test
significant:</p></li>
<li><p>Performing on peripheral genes:</p></li>
</ul>
<p><img src="figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-4-1.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/renga011/working_notes_liu_lab/blob/b646a9ea6a65d28c62539ff64339cf8f78b7d99a/docs/figure/2024-04-10_snp_summary.Rmd/unnamed-chunk-4-1.png" target="_blank">b646a9e</a>
</td>
<td>
renga011
</td>
<td>
2025-04-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Enrichment seen in <strong>peripheral immune signaling
pathways</strong></p></li>
<li><p>Likely driven by <strong>HLA region</strong>, which is very gene
dense. The SNP based COTA assigns the SNP to the nearest genes with a
cis pQTL. In such a gene dense region as HLA, you get multiple such
assignments, and these will lead to enrichment in the immune signaling
pathways, given all of them are of similar function. Hence, the
GO/KEGG/Reactome enrichment is not so informative.</p></li>
</ul>
</div>
<div id="what-next-1" class="section level3">
<h3>What next?</h3>
<ul>
<li>Too little enrichments being seen. Have to amp up the power of the
WES datasets. Search for a larger set or do meta-analysis over multiple
WES sets maybe?!</li>
<li>Many of the obvious genes known to be associated with AD aren’t
enriched in COTA. See what’s up with them? What is their trans p value
and WES p values. Make a table for these genes.</li>
</ul>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code></pre></div>
<pre><code>R version 4.4.1 (2024-06-14)
Platform: aarch64-apple-darwin20
Running under: macOS Ventura 13.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] igraph_2.1.4    dplyr_1.1.4     magrittr_2.0.3  gridExtra_2.3  
[5] ggplot2_3.5.1   workflowr_1.7.1

loaded via a namespace (and not attached):
 [1] sass_0.4.9        generics_0.1.3    stringi_1.8.7     hms_1.1.3        
 [5] digest_0.6.37     evaluate_1.0.3    grid_4.4.1        fastmap_1.2.0    
 [9] rprojroot_2.0.4   jsonlite_2.0.0    processx_3.8.6    whisker_0.4.1    
[13] ps_1.9.0          promises_1.3.2    httr_1.4.7        scales_1.3.0     
[17] jquerylib_0.1.4   cli_3.6.4         rlang_1.1.5       crayon_1.5.3     
[21] bit64_4.6.0-1     munsell_0.5.1     withr_3.0.2       cachem_1.1.0     
[25] yaml_2.3.10       parallel_4.4.1    tools_4.4.1       tzdb_0.5.0       
[29] colorspace_2.1-1  httpuv_1.6.15     vctrs_0.6.5       R6_2.6.1         
[33] lifecycle_1.0.4   git2r_0.36.2      stringr_1.5.1     bit_4.6.0        
[37] fs_1.6.5          vroom_1.6.5       pkgconfig_2.0.3   callr_3.7.6      
[41] pillar_1.10.1     bslib_0.9.0       later_1.4.1       gtable_0.3.6     
[45] glue_1.8.0        Rcpp_1.0.14       xfun_0.51         tibble_3.2.1     
[49] tidyselect_1.2.1  rstudioapi_0.17.1 knitr_1.50        farver_2.1.2     
[53] htmltools_0.5.8.1 rmarkdown_2.29    labeling_0.4.3    readr_2.1.5      
[57] compiler_4.4.1    getPass_0.2-4    </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
